### ISSUE TITLE
SDL2.pas: SDL_AUDIO_ISLITTLEENDIAN stack overflow

### ISSUE THREAD

**Issue by Chixpy** [PascalGameDevelopment/SDL2-for-Pascal#138]
Hello. I found another little mistake.

https://github.com/PascalGameDevelopment/SDL2-for-Pascal/blob/0fcaf6a5a93d2fd8b55726b6384ba3787b454728/units/sdl2.pas#L352-L355

It's calling itself recursively, when clearly we want to call `SDL_AUDIO_ISBIGENDIAN`

```pascal
function SDL_AUDIO_ISLITTLEENDIAN(x: Cardinal): Cardinal;
begin
  Result := not SDL_AUDIO_ISBIGENDIAN(x);
end;
``` 

Can I make a fast PR to fix it?

**Comment by suve** [comment]
Oh oops. Well that's embarrassing.

> Can I make a fast PR to fix it?

Sure. For small fixes like these, feel free to open a PR right away.

**Comment by suve** [comment]
Solved by PR #139. Thanks!
